Javascript/achievements.js

```diff
@@ -329,313 +327,532 @@ var areward182 = "Unlock Tier 8 Ant autobuy, and autobuy Scientia and Phylacteri
+    if (player.reincarnatenoaccelerator == true) {
+      achievementaward(62)
+    }
+    if (player.reincarnatenomultiplier == true) {
+      achievementaward(59)
+    }
+    if (player.reincarnatenocoinupgrades == true) {
+      achievementaward(67)
+    }
+    if (player.reincarnatenocoinorprestigeupgrades == true) {
+      achievementaward(68)
+    }
+    if (player.reincarnatenocoinprestigeortranscendupgrades == true) {
+      achievementaward(69)
+    }
+    if (player.reincarnatenocoinprestigetranscendorgeneratorupgrades == true) {
+      achievementaward(70)
+    }
+    if (reincarnationPointGain.greaterThanOrEqualTo(1)) {
+      achievementaward(50)
 
-        }
-        if (reincarnationPointGain.greaterThanOrEqualTo(1e5)) {
-            achievementaward(51)
+    }
+    if (reincarnationPointGain.greaterThanOrEqualTo(1e5)) {
+      achievementaward(51)
 
-        }
-        if (reincarnationPointGain.greaterThanOrEqualTo(1e30)) {
-            achievementaward(52)
+    }
+    if (reincarnationPointGain.greaterThanOrEqualTo(1e30)) {
+      achievementaward(52)
 
-        }
-        if (reincarnationPointGain.greaterThanOrEqualTo(1e200)) {
-            achievementaward(53)
+    }
+    if (reincarnationPointGain.greaterThanOrEqualTo(1e200)) {
+      achievementaward(53)
 
-        }
-        if (reincarnationPointGain.greaterThanOrEqualTo("1e1500")) {
-            achievementaward(54)
+    }
+    if (reincarnationPointGain.greaterThanOrEqualTo("1e1500")) {
+      achievementaward(54)
 
-        }
-        if (reincarnationPointGain.greaterThanOrEqualTo("1e5000")) {
-            achievementaward(55)
+    }
+    if (reincarnationPointGain.greaterThanOrEqualTo("1e5000")) {
+      achievementaward(55)
 
-        }
-        if (reincarnationPointGain.greaterThanOrEqualTo("1e7777")) {
-            achievementaward(56)
+    }
+    if (reincarnationPointGain.greaterThanOrEqualTo("1e7777")) {
+      achievementaward(56)
 
-        }
     }
+  }
 }
 
 
-function challengeachievementcheck(i,auto) {
-    var generatorcheck = Math.max(player.upgrades[101] + player.upgrades[102] + player.upgrades[103] + player.upgrades[104] + player.upgrades[105])
-if (i == 'one') {
-    if (player.challengecompletions.one > 0.5) {achievementaward(78)}
-    if (player.challengecompletions.one > 2.5) {achievementaward(79)}
-    if (player.challengecompletions.one > 4.5) {achievementaward(80)}
-    if (player.challengecompletions.one > 9.5) {achievementaward(81)}
-    if (player.challengecompletions.one > 19.5) {achievementaward(82)}
-    if (player.challengecompletions.one > 49.5) {achievementaward(83)}
-    if (player.challengecompletions.one > 74.5) {achievementaward(84)}
+function challengeachievementcheck(i, auto) {
+  var generatorcheck = Math.max(player.upgrades[101] + player.upgrades[102] + player.upgrades[103] + player.upgrades[104] + player.upgrades[105])
+  if (i == 'one') {
+    if (player.challengecompletions.one > 0.5) {
+      achievementaward(78)

```

Comment:
These can still be one-liners.
```js
if (player.challengecompletions.one > 0.5) achievementaward(78);
```

Reply:
Redid that method for around 200 line saves, still not doing one-liner, though.